app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::FactsController < ApplicationController
+    include AuthenticationCheck
+
+  before_action :is_user_logged_in
+  before_action :set_member, only: [:index, :show, :update, :destroy]
+  before_action :set_fact, only: [:show, :update, :destroy]
+  
+  # GET /members/:member_id/facts
+  def index
+    @member = Member.find(params[:member_id])
```

Comment:
Because you already execute this via the `before_action :set_member` a few lines above, this should be deleted.